### ISSUE TITLE
Proposal: iOS support

### ISSUE THREAD

**Issue by jaime-olivares** [Microsoft/vscode#10928]
I have searched for a similar proposal but couldn't find it here in github.

Per my understanding, VSCode is mainly intended to be a "powerful" editor. I think it has evolved to a full IDE, with compilation, linters, package managers, taskers and debugging capabilities, among others.

Going back to the basis, it seems that VSCode can be compiled for iOS platforms, specially the iPad Pro. The principal goal: code reviewing and eventual editing. No need of all the other stuff.

In summary, we would need the basic Monaco editor, with theme support, syntax highlighting and of course git capabilities. It seems node.js is also supported (see https://github.com/node-app). The whole application can be wrapped with Apache Cordova for easy installation and filesystem for the git source code.

Although all this can be done by any independent developer, it would be nice to have the Visual Studio Code branding and be aligned with its releases.

**Comment by joaomoreno** [comment]
This is a good project idea. But not within the VS Code scope.

Note that the [Monaco editor is open source](https://github.com/Microsoft/monaco-editor), in case someone wants to pick it up.

**Comment by johnjelinek** [comment]
This sounds like a fun weekend project. I've been using my iPad Pro to ssh into a server and vi my way to success, but maybe I can bring all the editing and compilation in the iPad itself. Sounds useful.
